theano/compile/nanguardmode.py

```diff
@@ -187,6 +200,47 @@ def compile_gpu_func(nan_is_error, inf_is_error, big_is_error):
             cuda_compile_failed = True
 
 
+def f_gpua_min(inp):
+    dt = inp.dtype
```

Comment:
can you use the same var name as everywhere else in the code and use dtype instead of dt